src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,125 @@
 package racingcar;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.*;
 
-public class Application {
+public class Application{
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        String inputName = System.console().readLine("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분");
+        String [] carNames = inputName.split("\\s*,\\s*");
+
+        String inputStr = System.console().readLine("시도할 회수를 입력하세요");
+        int inputCount = Integer.parseInt(inputStr);
+
+        checkError.checkCarName(carNames);
+
+        Car[] cars= new Car[carNames.length];
```

Comment:
java에서는 이러한 연속된 값을 다룰때 Collection을 사용합니다.

```suggestion
        List<Car> cars= new ArrayList();
```